spec/cocina/models/file_shared_examples.rb

```diff
@@ -0,0 +1,280 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.shared_examples 'it has file attributes' do
+  let(:item) { described_class.new(properties) }
+  let(:file_type) { 'http://cocina.sul.stanford.edu/models/file.jsonld' }
+  let(:properties) { required_properties }
```

Comment:
It would be helpful to have a comment somewhere describing what required properties are and who is supposed to provide them (perhaps referencing a line with an example).

Reply:
ok.

Reply:
I've added a block comment to the shared_examples which hopefully addresses your excellent suggestion.